index.js

```diff
@@ -0,0 +1,169 @@
+alert("Welcome to calculator");
+
+var typeCalculator = prompt("Choose (s)imple / (a)dvanced / (b)mi / (t)rip");
+console.log(typeCalculator);
+
+if (typeCalculator == "s"){
+
+	console.log("Simple")
+
+	alert("Welcome to simple calculator");
+
+	var stotal;
+
+	var sgetFirstNumber = prompt("Choose the first number");
+	console.log(sgetFirstNumber);
+
+	var soperateSum = prompt("Choose which operator, (1)Addition, (2)Subtraction, (3)Multiplication, (4)Division");
+	console.log(soperateSum);
+
+	if (soperateSum == "1"){
+
+		var sgetSecondNumber = prompt("Add");
```

Comment:
These prompts aren't really that clear what they want from the user. Should say something like "Please enter a second number to add". Try to keep this consideration of user experience in mind for the future.